doc/sphinx/azhelpgen/doc_source_map.json

```diff
@@ -1,72 +1,72 @@
-    "hdinsight": "src/azure-cli/azure/cli/command_modules/hdinsight/_help.py",
-    "iot": "src/azure-cli/azure/cli/command_modules/iot/_help.py",
-    "iotcentral": "src/azure-cli/azure/cli/command_modules/iotcentral/_help.py",
-    "keyvault": "src/azure-cli/azure/cli/command_modules/keyvault/_help.py",
-	"kusto": "src/azure-cli/azure/cli/command_modules/kusto/_help.py",
-    "lab": "src/azure-cli/azure/cli/command_modules/lab/_help.py",
-    "maps": "src/azure-cli/azure/cli/command_modules/maps/_help.py",
-    "monitor": "src/azure-cli/azure/cli/command_modules/monitor/_help.py",
-    "natgateway": "src/azure-cli/azure/cli/command_modules/natgateway/_help.py",
-    "network": "src/azure-cli/azure/cli/command_modules/network/_help.py",
-    "policy": "src/azure-cli/azure/cli/command_modules/resource/_help.py",
-    "policyinsights": "src/azure-cli/azure/cli/command_modules/policyinsights/_help.py",
-    "privatedns": "src/azure-cli/azure/cli/command_modules/privatedns/_help.py",
-    "provider": "src/azure-cli/azure/cli/command_modules/resource/_help.py",
-    "rdbms": "src/azure-cli/azure/cli/command_modules/rdbms/_help.py",
-    "redis": "src/azure-cli/azure/cli/command_modules/redis/_help.py",
-    "reservations": "src/azure-cli/azure/cli/command_modules/reservations/_help.py",
-    "resource": "src/azure-cli/azure/cli/command_modules/resource/_help.py",
-    "relay": "src/azure-cli/azure/cli/command_modules/relay/_help.py",
-    "role": "src/azure-cli/azure/cli/command_modules/role/_help.py",
-    "security": "src/azure-cli/azure/cli/command_modules/security/_help.py",
-    "signalr": "src/azure-cli/azure/cli/command_modules/signalr/_help.py",
-    "sql": "src/azure-cli/azure/cli/command_modules/sql/_help.py",
-    "sqlvm": "src/azure-cli/azure/cli/command_modules/sqlvm/_help.py",
-    "storage": "src/azure-cli/azure/cli/command_modules/storage/_help.py",
-    "tag": "src/azure-cli/azure/cli/command_modules/resource/_help.py",
-    "vm": "src/azure-cli/azure/cli/command_modules/vm/_help.py",
-    "vmss": "src/azure-cli/azure/cli/command_modules/vm/_help.py",
-    "servicefabric": "src/azure-cli/azure/cli/command_modules/servicefabric/_help.py",
-    "eventhubs": "src/azure-cli/azure/cli/command_modules/eventhubs/_help.py",
-    "servicebus": "src/azure-cli/azure/cli/command_modules/servicebus/_help.py",
-    "search": "src/azure-cli/azure/cli/command_modules/search/_help.py",
-    "managedapp": "src/azure-cli/azure/cli/command_modules/resource/_help.py",
-    "aks": "src/azure-cli/azure/cli/command_modules/acs/_help.py",
-    "deployment": "src/azure-cli/azure/cli/command_modules/resource/_help.py",
-    "functionapp": "src/azure-cli/azure/cli/command_modules/appservice/_help.py",
-    "lock": "src/azure-cli/azure/cli/command_modules/resource/_help.py",
-    "managedservices": "src/azure-cli/azure/cli/command_modules/managedservices/_help.py"
+    "az": "src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/_help.py",
```

Comment:
The reason there are so many edits to this file is because of a merge error. Please undo all changes that are not related to your module.